linkerd/transport-header/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,22 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use libfuzzer_sys::arbitrary::Arbitrary;
+
+#[derive(Debug, Arbitrary)]
+pub struct AllocatorMethod {
+    pub data: Vec<u8>,
+    pub port: u16,
+    pub protocol: bool,
+}
```

Comment:
nit, not important: I don't think any of this needs to be `pub`?
```suggestion
struct AllocatorMethod {
    data: Vec<u8>,
    port: u16,
    protocol: bool,
}
```